src/probnum/__init__.py

```diff
@@ -44,7 +45,17 @@
     statespace,
     utils,
 )
-from .randvars import RandomVariable, asrandvar
+from ._probabilistic_numerical_method import ProbabilisticNumericalMethod
+from .randvars import asrandvar
+
+# Public classes and functions. Order is reflected in documentation.
+__all__ = [
+    "asrandvar",
```

Comment:
is there a reason that this method remains in the probnum namespace, even though all the random variables do not?

Reply:
It's a deliberate choice (similar to `np.asarray`, `torch.as_tensor`) but I can be convinced to move it to randvars. Future functions which can move into the `probnum` namespace are `probnum.sum`, etc. which can operate on random variables / processes.

Reply:
I would move it to randvars, and find it a bit out of place here personally, but do not mind if you want to leave it in. You're opinion on this matter may be stronger than mine.